### PR TITLE
Wedding details styling km

### LINKED ISSUE #40: Styling for GuestList page

### PR DESCRIPTION
**Pull request by Kristiannmiller**
# Pull Request

## What's this PR do? 

- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing
- [X] Styling
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Description of changes?
- Adds basic styling to GuestList and Guest components
- Removes sidebar during photolist editing
- Adds StyledCard styled component to WeddingCard and GuestList components
- Adds image asset for empty guest list error handling
- Removes vestigial css replaced by styled components

## Any background context you want to provide? 
- Still need to address some responsive issues (mainly for height changes)

## What are the relevant tickets? 
* Closes #40

### CODE REVIEW

**Review by hannakim91 — APPROVED** [review]
This looks great - the guest list creation page is lot more user-friendly and interactive with the added styling!

File: src/components/App/styledComponents.styles.ts [hunk 1]
```diff
@@ -1,5 +1,84 @@
...
+    box-shadow: ${props => {
+      if (props.contents === 'wedding') {
+        return '1px 1px 20px inset #A8826F;'
+      } else {
+        return '1px 1px 20px #A8826F;'
+      }
+    }};
+  }
+`
+
```
Comment (ericberglund117) [inline src/components/App/styledComponents.styles.ts:81]: So I guess I didn't know enough about styled components. Pretty cool you use methods like that to determine styling

File: src/components/Guest/Guest.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  border: none;
+}
+.deleteButton:hover {
+  background-color: #f5f5f5;
+  color: #A8826F;
+  border: 2px solid #A8826F;
+}
+.deleteButton:focus {
+  outline: none;
+}
```
Comment (ericberglund117) [inline src/components/Guest/Guest.css:41]: I think this looks great!

File: src/components/Guest/Guest.tsx [hunk 3]
```diff
@@ -14,14 +15,15 @@ const Guest: React.FC<Props> = ({
 }) => {
   return (
     <article className="guestCard">
-      <h1>{guestName}</h1>
-      <h2>{phoneNumber}</h2>
+      <h1 className="guestInfo">{guestName}</h1>
+      <h2 className="guestInfo">{phoneNumber}</h2>
```
Comment (ericberglund117) [inline src/components/Guest/Guest.tsx:19]: I don't think will cause a merge conflict with my branch, but if it does, it's an easy fix

**Review by ericberglund117 — APPROVED** [review]
I appreciate you doing this. It makes the styling much easier in the long run

**Pull request merged**